### DISCUSSION TITLE
Error when adding Notion data source in the community version of Dify

### DISCUSSION THREAD

**Discussion by Bourne-alt** [langgenius/dify#7478] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

When I add a Notion data source on the community version of Dify, I encounter the following error when clicking the ‘Add Workspace’ button:
{"error":"invalid_request","error_description":"query failed validation: query.client_id should be a string or `undefined`, instead was `0`.","request_id":"6bb41008-4f72-41e5-b84a-825add13f49b"}

### 2. Additional context or comments

.env file parameters have already been configured:
NOTION_INTEGRATION_TYPE=internal 
NOTION_INTERNAL_SECRET=secret_******

**Comment by HoyiousXBFamily** [comment]
Finally, I solved this problem, if I guess right, I think your OS is MACOSX, so your Docker's Version is `Docker Desktop`. 

> If you are sure that your configuration information is right, but the `"query failed validation: query.client_id should be a string or undefined, instead was 0.",` the problem still exists. 

The only reason is that Dify didn't load your .env configuration right. And this is because you restarted Dify in Docker Client instead of Terminal. You should restart it in your terminal, dude. 

The code is:
(cd to your Dify Docker Path) insert
`docker compose down` to shut down Dify, and insert `docker compose up -d` to start Dify. And then .env will be loaded correctly.

**Reply by terry00lu** [reply]
I have  the same problem, an i did your step.  and it doesn't work.

**Reply by bbb324** [reply]
me either, still not working
<img width="570" alt="image" src="https://github.com/user-attachments/assets/df9532dd-06db-4a5a-a11c-28482bb8db18" />

**Reply by HoyiousXBFamily** [reply]
@bbb324 @terry00lu

Sorry, brothers. I’ve uninstalled Dify, so I apologize to both of you; I can’t respond to your latest messages right now. 😢

> I’m using a more convenient and flexible structure now.
